pkg/config/config.go

```diff
@@ -0,0 +1,98 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// SpotterConfig represents the Spotter configuration
+type SpotterConfig struct {
```

Comment:
this new config file only has these 5 variables? If yes, we can combine it with our config file `.spotter.yaml`, rebrand them both as `~/.spotter/config.yaml`

Reply:
Fixed